chore: created an issue template for Consul docs day (#11775)
hashicorp/consul: .github/ISSUE_TEMPLATE/docs_day.md

---
name: Consul docs day task
about: This is a HashiCorp internal documentation task for the purpose of the Consul docs day event.

> labels: ['type/docs', 'placeholder']

Reviewer: Change to:
```suggestion
labels: ['type/docs', 'event/consul-docs-day', `meta/maintainer-preferred`]
```

assignees: 
    - karl-cardenas-coding
    - jkirschner-hashicorp
    - trujillo-adam
---

body:
  - type: markdown
    attributes:
      value: |
        Thanks for taking the time to fill out this Consul docs event task request!
 - type: dropdown
    id: Category
    attributes:
      label: Category
      description: What category is this task for?
      options:
        - Add/Improve Examples
        - Structural Improvements and Cleanup

>         - Visual Aids

Author: ```suggestion
        - Visual Aids
        - Other
```

---
name: Consul docs day task
about: This is a HashiCorp internal documentation task for the purpose of the Consul docs day event.

> labels: ['consul', 'placeholder']

Reviewer: Maybe 
```suggestion
labels: ['type/docs', 'placeholder']
```

I'll send a Slack message to the community team about what to put instead of `placeholder`.

assignees: 
    - karl-cardenas-coding
    - jkirschner-hashicorp
    - trujillo-adam
---

body:
  - type: markdown
    attributes:
      value: |
        Thanks for taking the time to fill out this Consul docs event task request!
 - type: dropdown
    id: Category
    attributes:
      label: Category
      description: What category is this task for?
      options:
        - Add/Improve Examples
        - Structural Improvements and Cleanup

>         - Visual Aids

Reviewer: I think we should include an option for `Other`.

Reviewer: @karl-cardenas-coding : did this get implemented somehow? (I can't tell)

    validations:
      required: true
  - type: input
    id: Short Name
    attributes:
      label: Short Name
      description: Please provide a short name for this task. 
      placeholder: ex. PKI Overview
    validations:
      required: true
  - type: textarea
    id: Description
    attributes:
      label: Description

>       description: Please provide details about the task?

Reviewer: ```suggestion
      description: Please provide details about the task.
```

Author: It's not a question but more of an ask 😅

Reviewer: So should it be changed to a period then?

>       placeholder: Tell us what you see!

Reviewer: Should we have different placeholder text here?

```suggestion
      placeholder: What I'll change and why this is important
```

    validations:
      required: true

  - type: textarea
    id: link
    attributes:

>       label: Documentation page link

Reviewer: ```suggestion
      label: Documentation page link(s)
```

>       description: Please provide a link to the documentation page (if applicable)

Reviewer: ```suggestion
      description: Please provide link(s) to the documentation page(s) to be modified, if applicable
```

      placeholder: ex: https://www.consul.io/docs/connect/gateways#gateways
    validations:
      required: true

  - type: textarea
    id: logs
    attributes:
      label: Relevant log output
      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.

>       render: shell

Reviewer: What is this for? Should it be deleted?
